base_app/src/app/cmdsets/material-popup-component.html

```diff
@@ -19,8 +17,8 @@
                        [formControl]="inputControl"
                        [value]="control.value"
                        [matDatepicker]="myDatepicker"
-                       [title]="control.desc"
-                />
```

Comment:
I'd probably leave the "/>" to avoid additional changes to merge.  It also seems more readable to have the closing ">" or "/>" on a new line. I'm pretty sure that "/>" is always a valid substitute for ">" for HTML tags that don't require a closing tag. For instance, I started using "<br/>" instead of "<br>" after getting invalid XHTML warnings from my IDE.